lib/config/presets/internal/replacements.ts

```diff
@@ -689,6 +690,17 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'rome-to-biome': {
+    description: 'The rome project has been discontinued and biome was announced.',
+    packageRules: [
+      {
+        matchDatasources: ['npm'],
+        matchPackageNames: ['rome'],
+        replacementName: '@biomejs/biome',
+        replacementVersion: '1.0.0',
+      },
+    ],
+  },
```

Comment:
Can we add a link to official document for manual migration instruction here? If so, how?

Reply:
Put it as part of the `description` field. Possibly we already support markdown here, meaning you can make it a markdown link, otherwise the full URL is ok

Reply:
Thank you for your advice! Add in 2dc3e28.